core/base/src/main/java/alluxio/AlluxioURI.java

```diff
@@ -311,6 +311,10 @@ public boolean hasAuthority() {
     return !(mUri.getAuthority() instanceof NoAuthority);
   }
 
+  /**
+   * Gets the hash code for the {@link URI}.
+   * @return  the hash code for the existing {@link #mUri}
```

Comment:
Defines an empty line between summary and params

Reply:
Fixed.